tools/hooks/lint.rb

```diff
@@ -56,7 +56,7 @@ def run_rubocop(files)
 end
 
 def run_eslint_apps(files)
-  run("./node_modules/.bin/eslint -c .eslintrc.js #{files.join(' ')}", APPS_DIR)
+  run("./node_modules/.bin/eslint -c .eslintrc.js -f ./.eslintCustomMessagesFormatter.js #{files.join(' ')}", APPS_DIR)
```

Comment:
I don't love that this needs to be updated in two places, but I think fixing that is outside the scope of this PR